src/components/Discord.tsx

```
@@ -1,9 +1,107 @@
+    
+    useEffect(() => {
+        const fetchDiscordLink = async () => {
+            try {
+                const response = await fetch(import.meta.env.VITE_GET_DISCORD_URL);
+                if (!response.ok) {
+                    throw new Error(`HTTP ERROR! status: ${response.status}`);
+                }
+                const data = await response.json();
+                console.log(data);
+                setDiscordLink(data.discordLink);
+            } catch (err: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)